contracts/contracts/GrantRegistry.sol

```diff
@@ -0,0 +1,135 @@
+pragma solidity ^0.8.5;
+
+contract GrantRegistry {
+  // Grant ID - The hash of the grants meetadata
+  bytes32 public grantRegistryId;
+  // Metadata Pointer
+  string public metaPtr;
+
+  // Owner
+  address owner;
+
+  modifier onlyOwner() {
+    require(msg.sender == owner);
+    _;
+  }
+
+  // TODO: Only allow the owner (which is the curator) address to configure the contract
+  // while its in config mode, then switch to active. Once active, the owner can only
+  // close the registry (have a conversation around this approach)
+  constructor(
+    address _owner,
+    bytes32 _grantRegistryId,
+    string memory _metaPtr
+  ) {
+    owner = _owner;
+    //state = State.CONFIG;
+    grantRegistryId = _grantRegistryId;
+    metaPtr = _metaPtr;
+  }
+
+  // State
+  enum State {
+    CONFIG,
+    ACTIVE,
+    CLOSED
+  }
+  //State public state;
+
+  // Grant Pointer Object
+  // Mapping from GrantID to set of Owner Address, Payout Address, Metadata Pointer, Replaces Grant (optional)
+  // TODO: Implement safemath for avoiding overflows with ints
+  uint256 public grantCount = 0;
+  mapping(bytes32 => Grant) public grants;
+  bytes32[] public grantIds;
+
+  struct Grant {
+    bytes32 id;
+    address owner;
+    address payout;
+    string metaPtr;
+    State state;
+    bytes32 replaces;
+  }
+
+  function activate(bytes32 _id) external onlyOwner {
```

Comment:
nit: recommend name change to `activateGrant` for clarity.

Also recommend emitting an event.

Reply:
Sounds good.